scripts/legislation/africa/namibia_english.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^58. Short title and commencement/,$p' | \
+    sed -n '/^PART I/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function move_article_titles_above_article_bodies {
+  sed -E "s/^(\([0-9]+\).*)\. /\1.\n\n/" 
+}
+
+function add_newlines_before_headers_and_articles {
+  local stdin="$(</dev/stdin)"
+
+  if [ "$#" -ne 1 ] ; then
+    echo_usage_error "$*" '<language>'
+    return 1
+  fi
+  local language="$1"
+
+  local line_prefix_regular_expression
+  line_prefix_regular_expression="$(get_line_prefix_regular_expression $language)"
+  local return_code="$?"
+  if [ "$return_code" -ne 0 ] ; then return "$return_code" ; fi
+
+  local regular_expression="([A-Z][A-Za-z]+|\.) ${line_prefix_regular_expression}"
+
+  echo "$stdin" | sed -E ":start;s/${regular_expression}/\1\n\2 \3/;t start"
```

Comment:
You should be able to obviate the `move_article_titles_above_article_bodies` function by amending this line as follows:

```
echo "$stdin" | sed -E "s/${regular_expression}/\n\n\1\n\2/g"
```

Please let me know if you run into any problems with this.